yarn-project/end-to-end/src/fixtures/snapshot_manager.ts

```diff
@@ -64,8 +69,11 @@ type SnapshotEntry = {
 export function createSnapshotManager(
   testName: string,
   dataPath?: string,
-  config: Partial<AztecNodeConfig> = {},
-  deployL1ContractsArgs: Partial<DeployL1ContractsArgs> = { assumeProvenThrough: Number.MAX_SAFE_INTEGER },
+  config: Partial<SetupOptions> = {},
```

Comment:
Making it take the same options as setup so they are more aligned ( they are almost the same) 

( could probably be the same with a bit of massaging )